Baby MonitorTests/Modules/App Reset/ApplicationResetterTests.swift

```diff
@@ -0,0 +1,150 @@
+import XCTest
+@testable import BabyMonitor
+import RxSwift
+import RxTest
+
+class ApplicationResetterTests: XCTestCase {
+    
+    private let resetEventMessage = "{\"action\":\"RESET_KEY\"}"
+    
+    private var initialAppMode: AppMode!
+    private var initialIsSendingCryingsAllowed: Bool!
+    private var initialSelfPushNotificationsToken: String!
+    private var initialReceiverPushNotificationsToken: String!
+    
+    private var fakeMessageServer: MessageServerMock!
```

Comment:
I think the convention in the project is `messageServerMock` :)

Reply:
👍